firefox-ios/firefox-ios-tests/Tests/XCUITests/ReadingListTests.swift

```diff
@@ -12,7 +12,8 @@ class ReadingListTests: BaseTestCase {
         navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
         navigator.nowAt(BrowserTab)
         mozWaitForElementToNotExist(app.staticTexts["Fennec pasted from XCUITests-Runner"])
-        app.buttons["Reader View"].waitAndTap()
+        mozWaitForElementToExist(app.buttons["Reader View"])
```

Comment:
waitAndTap() was actually doing a mozWaitForElementToExists.. hopefully the tapOnApp will help here to tap on the element and make the Display Settings visible..
Let's monitor this closely.
Thanks!